README.md

```diff
@@ -33,6 +33,64 @@ My Button
 :clipboard: [React UI Documentation](https://github.com/OSCA-Kampala-Chapter/react-ui/tree/main/reactui-docs)
 :office: [Website](https://github.com/OSCA-Kampala-Chapter/react-ui/tree/main/website)
 
+
+## Build Documentation Website
```

Comment:
@Rishav1707 thank you for opening this pr...could we please have this content added under the [readme file](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/reactui-docs/README.md) for the [reactui-documentation.](https://github.com/OSCA-Kampala-Chapter/react-ui/tree/main/reactui-docs) 

This is to enable proper separation of concerns as this is directly guiding on how to set up and run that [documentation website](https://osca-kampala-chapter.github.io/react-ui/)